apps/system/test/unit/statusbar_test.js

```diff
@@ -658,6 +507,35 @@ suite('system/Statusbar', function() {
       assert.isTrue(Statusbar.element.classList.contains('fullscreen-layout'));
       assert.isTrue(stub.calledOnce);
     });
+
+    test('setAppearance maximized with fewIcons', function() {
+      hideIcons();
+      showIcons(5);
+      Statusbar.calculateMinimized();
```

Comment:
can you do the `calculateMinimized` in the setup instead? we shouldn't have to call it at a specific moment